_includes/markdown/Systems.md

```diff
@@ -252,3 +253,94 @@ Once tuning moves beyond these main items, performance improvements will be mino
+
+### Memcached
+
+Memcached is simpler than Redis and has fewer features. While Redis can be used as a full database, memcached is only a key-value store. This simplicity is by design and makes Memcached a very low maintenance tool. 
+
+#### High Availability
+
+Memcached can be installed on multiple servers that can combine into a memcached “pool”.  Keeping with the theme of simplicity, each memcached server in the pool knows nothing about the other instances in the pool and operates completely independently.  It is up to the client (in this case, WordPress and PHP) to distribute data across the pool of memcached servers however it sees fit.  Memcached performs no replication, failover, or connection balancing itself.  To distribute data evenly across multiple servers, the PHP extensions for memcached use a hashing strategy.  In this way, multiple PHP web servers can read and write to multiple memcached servers and all know exactly where each key is stored.  If one server fails, keys are distributed across the remaining memcached instances.  This can have a hit on performance (especially compared to replication scheme) as keys need to be regenerated to replace the missing data, but is by far the easiest type of distributed cache to setup.
+
+#### Item Size
+
+By default, memcached accepts items (key + value) of 1 MB or less. In most scenarios, this is fine and plenty of space. However, on larger WordPress sites or sites with a number of plugins, the “alloptions” array combining all autoloaded rows from the wp_options table can exceed 1 MB.  This array will be stored in memcached if memcached is in use for the object cache, unless it exceeds the memcached item size limit.  When this array is larger than the memcached item size, it can cause all sorts of odd issues and inconsistent performance.  
+
+Starting in memcached version 1.4.2, the item size is configurable via the `-I` option.  Setting the item size higher than 1 MB is not recommended unless necessary as memcached becomes less efficient as the item size increases, meaning it will take more memory to store the same amount of data.  If memory is available to accommodate a larger cache size, it is worth considering raising the item size to prevent the alloptions array from ever becoming a problem.  
+
+#### Connecting WordPress to Memcached
```

Comment:
I've had trouble in the past choosing the right one for WP Local Docker and I've seen people with the same problem. It might make sense to call out which one is the correct one to use with WP Local Docker.

Reply:
I have no idea which is the right one to use with WP Local Docker and really it should be compatible with both.  I think the documentation for wp local docker would be the better place to document this.